pkg/components/components.go

```diff
@@ -20,7 +20,7 @@ const (
 	SriovDpImageDefault             = "quay.io/kubevirt/cluster-network-addon-sriov-device-plugin:v2.0.0-1.git9a20829"
 	SriovCniImageDefault            = "quay.io/kubevirt/cluster-network-addon-sriov-cni:v1.1.0-1.git9e4c973"
 	KubeMacPoolImageDefault         = "quay.io/kubevirt/kubemacpool:v0.3.0"
-	NMStateStateHandlerImageDefault = "quay.io/nmstate/kubernetes-nmstate-state-handler:v0.1.0"
+	NMStateStateHandlerImageDefault = "quay.io/nmstate/kubernetes-nmstate-state-handler:v0.2.0.gitd5fe13b3"
```

Comment:
v0.2.0 is enough if it is the top of the tag, with no additional commits

Reply:
Done